x-pack/metricbeat/module/googlecloud/compute/_meta/data.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_index": "metricbeat-8.0.0-2019.11.27-000001",
+    "_id": "H4iRrm4B7vvv-udOwwFO",
+    "_version": 1,
+    "_score": null,
+    "_source": {
+        "@timestamp": "2019-11-27T20:30:00.000Z",
+        "ecs": {
+            "version": "1.2.0"
+        },
+        "host": {
+            "name": "mcastro"
+        },
+        "agent": {
+            "version": "8.0.0",
+            "type": "metricbeat",
+            "ephemeral_id": "cb56f25c-fc0e-4bbc-9c49-4551c1a97ab5",
+            "hostname": "mcastro",
+            "id": "7e36a073-1a32-4a94-b65b-4c7f971fb228"
+        },
+        "cloud": {
+            "availability_zone": "us-central1-a",
+            "account": {
+                "id": "elastic-metricbeat"
+            },
+            "provider": "googlecloud",
+            "instance": {
```

Comment:
Is there a separate API to get more info for each compute instance? For example the machine type, status and etc.

Reply:
Oh yes! And I'm actually using it already but I completely forgot to attach machine type too! Thanks for the heads up!